velox/experimental/cudf/exec/CudfHashAggregation.cpp

```diff
@@ -394,19 +396,19 @@ std::unique_ptr<cudf_velox::CudfHashAggregation::Aggregator> createAggregator(
     bool isGlobal) {
   // Companion function may be count_merge_extract or count_partial or others,
   // so use this to map
-  if (kind.rfind("sum", 0) == 0) {
+  if (boost::algorithm::ends_with(kind, "sum")) {
```

Comment:
As the comment explains, we also want to catch kind that starts with "sum". 

Also nit: you don't need a boost algorithm when you can use stl.

Reply:
changed the code to make sure the default behaviour is not changed.